src/effects/effects_for_liquidity_pools_request.rs

```diff
@@ -0,0 +1,145 @@
+use crate::models::{Order, Request};
+use crate::BuildQueryParametersExt;
+
+#[derive(Default)]
+pub struct EffectsForLiquidityPoolRequest {
```

Comment:
Ik mis hier nog documentatie rondom de `EffectsForLiquidityPoolRequest` zelf.

Net als bij `SingleAccountRequest<I>`, zie je dat Eline niet alleen documentatie heeft toegevoegd, maar ook referenties naar de Horizon API Endpoint, code examples die getest worden met cargo test (doctest).

Dit zou eigenlijk standaard voor alle publieke structs moeten zijn.

Zo mis ik dit ook voor `EffectsForAccountRequest` bijv. Zit niet in deze PR, maar mag er wel bij wat mij betreft.